app/server/__init__.py

```diff
@@ -118,11 +119,17 @@ def before_request():
 
     @app.after_request
     def after_request(response):
-            # Execute any async celery tasks
+        from server.utils import pusher
 
         if response.status_code < 300 and response.status_code >= 200:
             db.session.commit()
 
+        for transaction, queue in g.pending_transactions:
+            transaction.send_blockchain_payload_to_worker(queue=queue)
```

Comment:
`send_blockchain_payload_to_worker` sets the task uuid, but there's no commit (which i think is good - commits in the after_Request have killed us in the past)?

Reply:
I think we will require a commit in after_request eventually though, just to avoid the race condition in the other PR. I did add that in here, and it seems to work fine though.